src/components/input/Input.tsx

```diff
@@ -37,18 +40,22 @@ export class Input extends React.Component<IInputProps, any> {
   render() {
     const classes = ['input-wrapper'].concat(this.props.classes);
     const labelClasses = [].concat(this.props.labelClasses);
+    const innerInputClasses = [].concat(this.props.innerInputClasses);
```

Comment:
We have been using the package "classnNames" to generate classes for a little while. Might be a good idea to use it here.